requirements/dev.in

```diff
@@ -0,0 +1,21 @@
+-c prod.txt
+-e .
```

Comment:
```suggestion
```

This directory does not contain a Python package that we want to install in editable form.